tests/importer/test_importer.py

```diff
@@ -18,10 +18,11 @@ def test_modify_documents(importer_test_data):
 
     importer = Importer(json_data[0], "springer")
     report = importer.import_record()
-    assert report["updated_document"]
+    assert report["document_json"]
+    assert report["action"] == "update"
 
     updated_document = document_cls.get_record_by_pid(
-        report["updated_document"]["pid"])
+        report["document_json"]["pid"])
     # wait for indexing
     time.sleep(1)
```

Comment:
Why is it 1 second wait for indexing? Could it take more?

Reply:
it is usually 1-2s, it never takes more than that, it was verified on different machines (otherwise the test wouldn't pass)

Reply:
What about using ES `flush` to wait instead?